specification/loadtestservice/data-plane/Microsoft.LoadTestService/preview/2023-04-01-preview/loadtestservice.json

```diff
@@ -2037,6 +2037,14 @@
         "optionalLoadTestConfig": {
           "$ref": "#/definitions/OptionalLoadTestConfig",
           "description": "Configuration for quick load test"
+        },
+        "regionConfiguration": {
```

Comment:
since we already have `optionalLoadTestConfig`
can we name it similarly: `regionalLoadTestConfig` so that the api naming remains consistent across.

Reply:
"regional" would imply the configuration is specific to a region IMO